core/src/code.rs

```diff
@@ -242,16 +240,17 @@ pub enum CodeError {
 }
 
 /// Contains instrumented binary code of a program and initial memory size from memory import.
-#[derive(Clone, Debug, Decode, Encode, PartialEq, Eq, TypeInfo)]
+#[derive(Clone, Debug, PartialEq, Eq)]
 pub struct Code {
     /// Code instrumented with the latest schedule.
     code: Vec<u8>,
     /// The uninstrumented, original version of the code.
-    raw_code: Vec<u8>,
+    original_code: Vec<u8>,
     /// Exports of the wasm module.
     exports: BTreeSet<DispatchKind>,
+    /// Static pages count from memory import.
     static_pages: WasmPage,
-    #[codec(compact)]
```

Comment:
I'm curious doesn't it break backward compatibility? Namely db storage

Reply:
why you removed?

Reply:
resolved